htsinfer/get_library_stats.py

```diff
@@ -88,7 +88,8 @@ def fastq_get_stats_read_length(fastq: Path) -> Tuple[
             FileProblem: Could not process FASTQ file.
         """
         LOGGER.debug(
```

Comment:
Isn't this sort of the same level of information that you now log via `info` instead of `debug`? :)

Reply:
Hm, there's actually already a log message in line 54, this is sort of redundant. Wouldn't it make sense to keep it `debug` for specifically the `fastq_get_stats_read_length` function?